packages/gafl-webapp-service/src/pages/licence-details/licence-start-time/route.js

```diff
@@ -9,7 +9,9 @@ const minHour = permission =>
     .tz(SERVICE_LOCAL_TIME)
     .isSame(moment(), 'day')
     ? moment()
+      .add(1, 'hour')
```

Comment:
Not sure I understand why we're adding an hour here? Should this be using moment-timezone instead?

Reply:
See above - the timezone is only used to check for start-date = today

Reply:
moment() give the local time

Reply:
moment() gives system time. This will likely not work when built into a docker container.